src/Services/EmailCredentialService.php

```diff
@@ -0,0 +1,100 @@
+<?php
+/**
+ * This file is part of the PHP Rebilly API package.
+ *
+ * (c) 2015 Rebilly SRL
+ *
+ * For the full copyright and license information, please view the LICENSE.md
+ * file that was distributed with this source code.
+ */
+
+namespace Rebilly\Services;
+
+use ArrayObject;
+use JsonSerializable;
+use Rebilly\Entities\EmailCredential;
+use Rebilly\Http\Exception\NotFoundException;
+use Rebilly\Http\Exception\UnprocessableEntityException;
+use Rebilly\Paginator;
+use Rebilly\Rest\Collection;
+use Rebilly\Rest\Service;
+
+/**
+ * Class EmailCredentialService
+ *
+ * @author Arman Tuyakbayev <arman.tuyakbayev@rebilly.com>
+ * @version 0.1
+ */
+final class EmailCredentialService extends Service
+{
+    /**
+     * @param array|ArrayObject $params
+     *
+     * @return EmailCredential[][]|Collection[]|Paginator
+     */
+    public function paginator($params = [])
+    {
+        return new Paginator($this->client(), 'email-credentials', $params);
+    }
+
+    /**
+     * @param array|ArrayObject $params
+     *
+     * @return EmailCredential[]|Collection
+     */
+    public function search($params = [])
+    {
+        return $this->client()->get('email-credentials', $params);
+    }
+
+    /**
+     * @param string $credentialId
+     * @param array|ArrayObject $params
+     *
+     * @throws NotFoundException The resource data does exist
+     *
+     * @return EmailCredential
+     */
+    public function load($credentialId, $params = [])
+    {
+        return $this->client()->get('email-credentials/{credentialId}', ['credentialId' => $credentialId] + (array) $params);
```

Comment:
I believe it shouldn't. First argument `$credentialId` is mandatory and important, and shouldn't be overridden.